docs/source/privacy/index.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Data Collection"
+sidebar_title: "Data Collection"
+description: "Information about data collected by Rover"
+---
+
+By default, Rover collects some anonymous usage data to help better understand the needs of our users. 
```

Comment:
"
By default, Rover collects anonymous usage data to help us improve the tool.

To opt out of data collection, set the `APOLLO_TELEMETRY_DISABLED` environment variable to `1` in each environment where you use Rover.
"

And then we can cut the "Opt Out" section at the bottom